docs/technical/resolve-merge-conflicts.md

```diff
@@ -83,7 +85,8 @@ It will look like this:
 
 ## Resolve conflicts
 
-Since this pull request does not modify the `package.json` file it is safe to fast forward the changes from `origin/main`:
+Since this pull request does not modify the `package.json` file it is safe to fast forward the changes from `origin/insert-default-branch`:
```

Comment:
on GitHub `HEAD` redirects to the default branch. This is also through for git. It would be confusing for new contributors to see `insert-default-branch` especially since most will lean towards copy and pasting. 

Replace `insert-default-branch` with `HEAD`. Also, put a note similar to what I mentioned about `HEAD` redirecting to the default branch, and I think that would cover the bases and make this mergeable.